### PR TITLE
Maintainer announcement update (to README, etc.)

### PR DESCRIPTION
**Pull request by sukritsingh**
Starting a PR where I'm updating the MDTraj README to indicate the transition to new maintainers (and welcoming continued support/contributions from folks). Thoughts/edits/discussions welcome!

For now at least the README is the only change made. I may have to dig around the docs a bit to ensure that appropriate contact info is placed there as well.

### CODE REVIEW

**Comment by sukritsingh** [comment]
@mattwthompson you may have thoughts here? Happy to add you or something since you've been so active as of late

**Review by mattwthompson — APPROVED** [review]
Great news! Is this directly funded, or just sorta swinging under the umbrella of F@H (and I assume the hand-off came from @rmcgibbo in some capacity)?

These changes look good to me - I don't mind being mentioned and don't really care either way. My only suggestion, which I assume is already the intent, is that the note is removed altogether in a few weeks once the repo gets some TLC.

**Comment by sukritsingh** [comment]
Sorta swinging under the umbrella of F@H for now - Xuhui is also working on taking over MSMBuilder and bringing that back up to speed so there's sort of a two pronged effort to get these FAH-relevant libraries back to order. I wasn't there for the handoff directly but Xuhui and Greg Bowman were given access power by @rmcgibbo to my knowledge (and Xuhui gave me powers in turn)! 

> My only suggestion, which I assume is already the intent, is that the note is removed altogether in a few weeks once the repo gets some TLC.

Hah yes the intention is that we a) put this note up for a bit, b) announce on any FAH social channels and c) remove the note after a couple months once we've cleaned up more.

> These changes look good to me - I don't mind being mentioned and don't really care either way.

Whatever you're comfortable with! You've put effort in so whatever you'd like to as appropriate acknowledgement!

File: README.md
Comment (mattwthompson) [inline README.md:10]: Feel free to mention me somewhere in here, then, thanks

Comment (sukritsingh) [reply README.md]: I've added you into the latest commit!

**Comment by sukritsingh** [comment]
Added you in! I'm going to quickly email Xuhui and Greg to get their eyes on this announcement. I'll merge in 24 hours if nothing else comes up.

**Comment by peastman** [comment]
My only comment is to say I'm really glad it will be actively maintained again.  It's a great library.

**Comment by sukritsingh** [comment]
Got the go-ahead from both! Merging it now.

**Comment by rmcgibbo** [comment]
Cheers all! I’m happy to see this, and to hand off the torch to such a capable crew!

-Robert

**Pull request merged**
